--- ./src/compat.h.orig	2016-01-07 10:13:09.000000000 +0100
+++ ./src/compat.h	2016-01-07 10:07:28.000000000 +0100
@@ -67,4 +67,38 @@
 
 #endif
 
+#if ENABLE_INOTIFY
+#else
+
+#define IPTOS_DSCP_MASK         0xfc
+#define IPTOS_DSCP(x)           ((x) & IPTOS_DSCP_MASK)
+#define IPTOS_DSCP_AF11         0x28
+#define IPTOS_DSCP_AF12         0x30
+#define IPTOS_DSCP_AF13         0x38
+#define IPTOS_DSCP_AF21         0x48
+#define IPTOS_DSCP_AF22         0x50
+#define IPTOS_DSCP_AF23         0x58
+#define IPTOS_DSCP_AF31         0x68
+#define IPTOS_DSCP_AF32         0x70
+#define IPTOS_DSCP_AF33         0x78
+#define IPTOS_DSCP_AF41         0x88
+#define IPTOS_DSCP_AF42         0x90
+#define IPTOS_DSCP_AF43         0x98
+#define IPTOS_DSCP_EF           0xb8
+
+#define IPTOS_CLASS_MASK        0xe0
+#define IPTOS_CLASS(class)      ((class) & IPTOS_CLASS_MASK)
+#define IPTOS_CLASS_CS0         0x00
+#define IPTOS_CLASS_CS1         0x20
+#define IPTOS_CLASS_CS2         0x40
+#define IPTOS_CLASS_CS3         0x60
+#define IPTOS_CLASS_CS4         0x80
+#define IPTOS_CLASS_CS5         0xa0
+#define IPTOS_CLASS_CS6         0xc0
+#define IPTOS_CLASS_CS7         0xe0
+
+#define IPTOS_CLASS_DEFAULT     IPTOS_CLASS_CS0
+
+#endif
+
 #endif /* COMPAT_H */
