--- ./configure.orig	2016-01-07 21:49:27.000000000 +0100
+++ ./configure	2016-01-07 21:49:13.000000000 +0100
@@ -662,7 +662,7 @@
 case $host_cpu in
     i*86)
         ARCH="X86"
-        AS="${AS-yasm}"
+        AS="yasm"
         AS_EXT=".asm"
         ASFLAGS="$ASFLAGS -DARCH_X86_64=0 -I\$(SRCPATH)/common/x86/"
         if [ $compiler = GNU ]; then
@@ -703,7 +703,7 @@
         ;;
     x86_64)
         ARCH="X86_64"
-        AS="${AS-yasm}"
+        AS="yasm"
         AS_EXT=".asm"
         ASFLAGS="$ASFLAGS -DARCH_X86_64=1 -I\$(SRCPATH)/common/x86/"
         [ $compiler = GNU ] && CFLAGS="-m64 $CFLAGS" && LDFLAGS="-m64 $LDFLAGS"
